test/runtime/fqdn.go

```diff
@@ -122,9 +126,9 @@ var _ = Describe("RuntimeFQDNPolicies", func() {
 		}
 
 		ciliumOutisdeImages = map[string]string{
-			WorldHttpd1: helpers.HttpdImage,
-			WorldHttpd2: helpers.HttpdImage,
-			WorldHttpd3: helpers.HttpdImage,
+			OutsideHttpd1: helpers.HttpdImage,
```

Comment:
Since we're touching this code, could you also fix the spelling of `Outisde` just above?